### ISSUE TITLE
Error when creating postgres 9.4 databases and users

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-postgresql#15]
_This issue was originally opened by @kshcherban as hashicorp/terraform#15620. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

Terraform fails to create databases and users for older Postgres versions.

Terraform 0.9.11, RDS Postgres 9.4.7.

Code:
```
resource "postgresql_role" "custom_auction_channel" {
  name     = "custom_auction_channel"
  login    = true
  password = "${data.vault_generic_secret.custom_auction_channel.data["password"]}"
}
```

```
* postgresql_role.custom_auction_channel: 1 error(s) occurred:

* postgresql_role.custom_auction_channel: Error creating role custom_auction_channel: pq: unrecognized role option "nobypassrls"
``` 

Database creation error:
```
* postgresql_database.rds_postgres_master1_databases.101: Error creating database "custom_auction_channel": pq: syntax error at or near ""UTF8""
```
Seems like relates to hashicorp/terraform#11273

**Comment by scholzie** [comment]
I'm experiencing this as well, attempting to apply to postgres 9.3.14

```
1 error(s) occurred:

* postgresql_role.bacom-read-only: 1 error(s) occurred:

* postgresql_role.bacom-read-only: Error creating role ro_user: pq: unrecognized role option "nobypassrls"
```

```
resource "postgresql_role" "bacom-read-only" {
  provider = "<redacted>"

  create_database           = false
  login                     = true
  name                      = "ro_user"
  password                  = "<redacted>"
  replication               = false
  superuser                 = false
}
```

Is there any known work around for this?

**Comment by madmod** [comment]
I also have this issue.

**Comment by cyrilgdn** [comment]
This is resolved since 0.1.1 thanks to https://github.com/terraform-providers/terraform-provider-postgresql/pull/12
